README.md

```diff
@@ -29,6 +31,23 @@ Amplify.configure(awsconfig);
   })
 ```
 
+#### Using AmplifyGeocoder with [maplibre-gl-geocoder](https://github.com/maplibre/maplibre-gl-geocoder)
+
+```js
+import Amplify from "aws-amplify";
+import awsconfig from './aws-exports';
+import maplibregl from "maplibre-gl";
+import MaplibreGeocoder from "@maplibre/maplibre-gl-geocoder";
+import "@maplibre/maplibre-gl-geocoder/dist/maplibre-gl-geocoder.css";
+
+Amplify.configure(awsconfig);
+...
+  const geocoder = new MaplibreGeocoder(AmplifyGeocoder, {
+    maplibregl: maplibregl,
+  });
+  map.addControl(geocoder);
```

Comment:
From this example, does it make sense for s/AmplifyGeocoder/AmplifyGeocoderAPI/ ?

Reply:
Do you mean renaming AmplifyGeocoder to AmplifyGeocoderAPI? I think that will probably be more clear so I can update it

Reply:
yeah sorry. Exactly that.